### PR TITLE
add ma endpoints

### PR DESCRIPTION
**Pull request by zsoltn**
please integrate the ModelArts devenv + models endpoint.

### CODE REVIEW

**Comment by vladimirhasko** [comment]
recheck

File: epmon/config.yaml [hunk 1]
```diff
@@ -194,6 +194,12 @@ elements:
       - /
       - /kms/list-keys
       - /kms/user-quotas
+  modelarts:
```
Comment (vladimirhasko) [inline epmon/config.yaml:197]: in the metric processor config there are also defined references for modelartsv2
so either add the epmon queries for modelartsv2 or adjust flag and health metrics to not refer to modelartsv2

**Review by vladimirhasko — CHANGES_REQUESTED** [review]

File: epmon/config.yaml (hunk 1 above)
Comment (zsoltn) [reply epmon/config.yaml:197]: added modelartsv2 endpoints. Thanks @vladimirhasko.

**Review by vladimirhasko — CHANGES_REQUESTED** [review]
Please add also modelartsv2 in config.yaml for eu-de so these queries can be triggered in respective region

**Comment by zsoltn** [comment]
> Please add also modelartsv2 in config.yaml for eu-de so these queries can be triggered in respective region

@vladimirhasko: modelartsv2 added to config.yaml

**Review by vladimirhasko — APPROVED** [review]

**Pull request merged**